larky/src/main/java/com/verygood/security/larky/modules/CerebroModule.java

```diff
@@ -0,0 +1,20 @@
+package com.verygood.security.larky.modules;
+
+import com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.TextAnalyzerModule;
```

Comment:
Do we really need this lengthy namespace? Why .text.analyzer ? Why not just analyzers.TextAnalyzerModule or analyzers.TextPIIScanner ?

Reply:
The idea is to structure it by use-cases, so when I go to `.text` I see everything related to intelligent data processing of text, it may be: analyzers, anonymizers(automatic redaction attempt), etc.
And the same may go for `.image.analyzer`, `.audio.analyzer`, etc.